pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -2191,6 +2191,30 @@ var _ = Describe("Manager", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(domStats).ToNot(BeNil())
 		})
+		Context("on failed GetAllDomainStats", func() {
```

Comment:
can you extract this from the ""on successful .." context?

Reply:
done